backend/src/routes/userRoute.test.js

```diff
@@ -0,0 +1,35 @@
+const rewire = require("rewire")
+const userRoute = rewire("./userRoute")
+const sendUserJson = userRoute.__get__("sendUserJson")
```

Comment:
If you have to 'break' into the production code to get at something it's often a signal of a design smell.

In addition, it's not what you're testing. The tests should describe the intention of what's being tested and the "1", "2" and `toMatchSnapshot` don't do this. Is it the mapping or response behaviour (http statuses). If that's clear, that's being described? If that's clearer, it will make clear what the design smell. is

Reply:
I should put the sendUserJson in it's own JS file